openreview/openreview.py

```diff
@@ -298,92 +178,51 @@ def get_notes(self, id=None, forum=None, invitation=None, replyto=None, tauthor=
             params['trash']=True
 
         response = requests.get(self.notes_url, params=params, headers=self.headers)
-        
+        response = self.__handle_response(response)
+
+        for n in response.json()['notes']:
+            note = Note.from_json(n)
+            notes.append(note)
+        notes.sort(key=lambda x: x.forum)
```

Comment:
Sort outside the loop, btw the notes service returns the notes sorted by date. I don't why you want to sort by forum